projects/plugins/jetpack/_inc/lib/class.color.php

```diff
@@ -828,56 +811,66 @@ public function toGrayscale() {
 	}
 
 	/**
-	 * Transform -- Get hue-incremented color, by 180.
+	 * Transform -- To the complementary color.
+	 *
+	 * The complement is the color on the opposite side of the color wheel, 180° away.
 	 *
-	 * @return Jetpack_Color
+	 * @return $this
 	 */
 	public function getComplement() {
 		return $this->incrementHue( 180 );
 	}
 
 	/**
-	 * Transform -- Get hue-incremented color, 30 steps at a time, with a base of 180.
+	 * Transform -- To an analogous color of the complement.
 	 *
-	 * @param int $step How many times we should increment hue by 30.
+	 * @param int $step Pass `1` or `-1` to choose which direction around the color wheel.
 	 *
-	 * @return Jetpack_Color
+	 * @return $this
 	 */
 	public function getSplitComplement( $step = 1 ) {
 		$incr = 180 + ( $step * 30 );
 		return $this->incrementHue( $incr );
 	}
 
 	/**
-	 * Transform -- Get hue-incremented color, 30 steps at a time.
+	 * Transform -- To an analogous color.
 	 *
-	 * @param int $step How many times we should increment hue by 30.
+	 * Analogous colors are those adjacent on the color wheel, separated by 30°.
 	 *
-	 * @return Jetpack_Color
+	 * @param int $step Pass `1` or `-1` to choose which direction around the color wheel.
+	 *
+	 * @return $this
 	 */
 	public function getAnalog( $step = 1 ) {
 		$incr = $step * 30;
 		return $this->incrementHue( $incr );
 	}
 
 	/**
-	 * Transform -- Get hue-incremented color, 60 steps at a time.
+	 * Transform -- To a tetradic (rectangular) color.
+	 *
+	 * A rectangular color scheme uses a color, its complement, and the colors 60° from each.
+	 * This transforms the color to its 60° "tetrad".
 	 *
-	 * @param int $step How many times we should increment hue by 60.
+	 * @param int $step Pass `1` or `-1` to choose which direction around the color wheel.
 	 *
-	 * @return Jetpack_Color
+	 * @return $this
 	 */
 	public function getTetrad( $step = 1 ) {
 		$incr = $step * 60;
 		return $this->incrementHue( $incr );
 	}
 
 	/**
-	 * Transform -- Get hue-incremented color, 120 steps at a time.
+	 * Transform -- To a triadic color.
+	 *
+	 * A triadic color scheme uses three colors evenly spaced (180°) around the color wheel.
```

Comment:
Looks like I had a typo in my suggestion :(
```suggestion
	 * A triadic color scheme uses three colors evenly spaced (120°) around the color wheel.
```